operation/periodic-task/prefect/workflow.py

```
@@ -0,0 +1,34 @@
+        raise ValueError("Database URL environment variable is not set: CRATEDB_SQLALCHEMY_URL")
+    engine = sa.create_engine(database_url, echo=True)
+
+    try:
+        with engine.connect() as conn:
+            result = conn.execute(sa.text("SELECT 42"))
+            record = result.one()
+            print(record)
+            # Return the results of your task to have it stored
+            # and accessible on the web UI.
+            return record
+        logger.info("Fetched %s database rows", len(sales))
```

Ruff (0.15.12)

[error] Undefined name `sales`

(F821)